src/views/ProfileView.jsx

```diff
@@ -1,5 +1,5 @@
 import React from "react";
-import { fetchBackend } from "../state/fetch-backend";
+import fetchBackend from "../state/fetch-backend";
```

Comment:
Any custom hook should start with `useXxxx`name

Reply:
I've updated this so that FetchBackend is a component, rather than a custom hook. Let me know if I should change it back (and then change the syntax)